src/class.js

```diff
@@ -5,10 +5,21 @@
 // Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
 // for a potential password that will be compared to the `password` property.
 // Return true if the potential password matches the `password` property.  Otherwise return false.
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
+}
 
+User.prototype.comparePasswords = function (currentPassword) {
+  if (currentPassword === this.password) {
```

Comment:
Just wanted to point out that you can shorten this to `return currentPassword === this.password;`. It's not wrong to do it the way you did, just letting you know about some other ways to do it.